Source/carthage/Archive.swift

```diff
@@ -61,10 +94,11 @@ public struct ArchiveCommand: CommandType {
 			.collect()
 			.flatMap(.Merge) { paths -> SignalProducer<(), CarthageError> in
 				if paths.isEmpty {
-					return SignalProducer(error: CarthageError.InvalidArgument(description: "Could not find any copies of \(options.frameworkName).framework. Make sure you're in the project’s root and that the framework has already been built using 'carthage build --no-skip-current'."))
+					let error = CarthageError.InvalidArgument(description: "Could not find any copies of \(frameworks). Make sure you're in the project’s root and that the frameworks has already been built using 'carthage build --no-skip-current'.")
```

Comment:
Also, it'd be nice to join the frameworks with commas so we don't get the `[]`s in the output.